PhysicsTools/JetMCAlgos/python/AK4PFJetsMCFlavourInfos_cfi.py

```diff
@@ -4,7 +4,7 @@
     jets                     = cms.InputTag("ak4PFJets"),
     bHadrons                 = cms.InputTag("selectedHadronsAndPartons","bHadrons"),
     cHadrons                 = cms.InputTag("selectedHadronsAndPartons","cHadrons"),
-    partons                  = cms.InputTag("selectedHadronsAndPartons","algorithmicPartons"),
+    partons                  = cms.InputTag("selectedHadronsAndPartons","physicsPartons"),
```

Comment:
should the same be applied to other available instances of JetFlavourClustering in this package?
There is one in AK5